### PR TITLE
Update feature_request.md

### PR DESCRIPTION
**Pull request by igordmn**
`about` can't be blank

### CODE REVIEW

**Comment by eymar** [comment]
Do we have a label "feature request"?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature request
-about: ''
+about: 'Suggest an idea for improving Compose Multiplatform'
 title: ''
 labels: ''
```
Comment (eymar) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
labels: 'enhancement'
```

Comment (igordmn) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Yes, we have it.

Also, changed name and about to the description of this tag

Comment (igordmn) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Also added `bug` to bug reports

**Pull request merged**